src/rpc/rawtransaction.cpp

```diff
@@ -833,29 +832,13 @@ UniValue signrawtransaction(const UniValue &params, bool fHelp)
             else if (boost::iequals(s, "ANYONECANPAY"))
                 nHashType |= SIGHASH_ANYONECANPAY;
             else if (boost::iequals(s, "FORKID"))
-            {
-                pickedForkId = true;
                 nHashType |= SIGHASH_FORKID;
-            }
-            else if (boost::iequals(s, "NOFORKID"))
-            {
-                pickedForkId = true;
-                nHashType &= ~SIGHASH_FORKID;
-            }
```

Comment:
I would like to keep the ability to sign a bitcoin-style transaction using signrawtransaction.  This may be useful by some people for cross-chain stuff.